sexp.c

```diff
@@ -1873,6 +1874,7 @@ sexp sexp_make_fileno_op (sexp ctx, sexp self, sexp_sint_t n, sexp fd, sexp no_c
   if (!sexp_exceptionp(res)) {
     sexp_fileno_fd(res) = sexp_unbox_fixnum(fd);
     sexp_fileno_openp(res) = 1;
+    sexp_fileno_count(res) = 0;
```

Comment:
Strange, `sexp_try_alloc` should zero out the whole chunk, so this shouldn't be necessary.